audbackend/core/backend/base.py

```diff
@@ -206,6 +211,32 @@ def _create(
         """
         raise NotImplementedError()
 
+    @classmethod
+    def create(
+        cls,
+        host: str,
+        repository: str,
+    ) -> Self:
+        r"""Create repository.
+
+        Creates ``repository``
+        located at ``host``
+        on the backend.
+
+        Args:
+            host: host address
+            repository: repository name
+
+        Raises:
+            BackendError: if an error is raised on the backend,
+                e.g. repository exists already
+                or cannot be created
+
+        """
+        backend = cls(host, repository)
+        utils.call_function_on_backend(backend._create)
+        return backend
```

Comment:
I think we have recently changed `audbackend.create()` to not (officially) return the `backend` object and only kept the return argument for backward compatibility. The argument was that in your code you should not do:

```python
backend = MyBackend.create()
# now do something with backend
```

Because running the code again will fail (since the repository now exists).

If we decide to switch back to returning the backend, we should document it in the `Returns` section of the docstring.

Reply:
Good point, I changed it to not return the backend (and removed `typing_extensions` from the package dependencies).
Until #199 is implemented, our `SQLite` example in the developer docs does then no longer work (don't know why it worked before with the returned backend ;) ). As stated in the description of this pull request, I uncommented and propose to undo those changes and update the example when we solve #199.

Reply:
It worked before, because we have `self._db = sl.connect(self._path)` inside `_create()`, which returned then a backend with an already connected database. But this also highlights, that it was not a good idea by me to add a return value to `create()` as accessing and leaving a backend should indeed be handled by #199.